bindings/gumjs/gumv8scope.h

```diff
@@ -115,4 +115,14 @@ class ScriptUnlocker
   v8::Unlocker unlocker;
 };
 
+class GumV8InterceptorIgnoreScope
+{
+public:
+  GumV8InterceptorIgnoreScope ();
```

Comment:
It's tempting to require `GumV8Core * core` here so we don't need to `obtain ()` and `unref ()` the `Interceptor`, and since `core` is available everywhere this is used. What do you think?

Reply:
let's go for it!

Reply:
hmm apparently we have it everywhere except in the original birthplace of `GumV8InterceptorIgnoreScope`, doesn't look like in `gumv8platform.cpp` we have a reachable `core`?

like:
```c++
GumV8PageAllocator::AllocatePages (void * address,
                                   size_t length,
                                   size_t alignment,
                                   Permission permissions)
{
  GumV8InterceptorIgnoreScope interceptor_ignore_scope;
...
```